src/lib/runtime_config/runtime_config.ml

```diff
@@ -1305,10 +1290,7 @@ module Daemon = struct
   let gen =
     let open Quickcheck.Generator.Let_syntax in
     let%bind txpool_max_size = Int.gen_incl 0 1000 in
-    let%bind zkapp_proof_update_cost = Float.gen_incl 0.0 100.0 in
-    let%bind zkapp_signed_single_update_cost = Float.gen_incl 0.0 100.0 in
-    let%bind zkapp_signed_pair_update_cost = Float.gen_incl 0.0 100.0 in
-    let%bind zkapp_transaction_cost_limit = Float.gen_incl 0.0 100.0 in
+    let%bind max_zkapp_segment_per_transaction = Int.gen_incl 0 50 in
```

Comment:
Unsure about these generators

Reply:
I am as well. The old generators could generate a config with `0` cost on the three zkapp components. I think that would mean no limit on the number of zkapp proof or signed updates.

Where are they used?